.upstream-tests/test/CMakeLists.txt

```diff
@@ -8,6 +8,19 @@ endmacro()
 
 set(LIBCUDACXX_HIGHEST_COMPUTE_ARCH 90)
 
+# Initialize the libcu++ target arches from the CMake property. If both are set,
+# favor the libcu++ variable.
+if (DEFINED CMAKE_CUDA_ARCHITECTURES)
+  if (NOT DEFINED LIBCUDACXX_COMPUTE_ARCHS)
+    list(JOIN CMAKE_CUDA_ARCHITECTURES " " LIBCUDACXX_COMPUTE_ARCHS)
```

Comment:
These should not be related. The libcudacxx is semantically different, and only used for advanced usecases. Leave it alone in this patch.